src/HCaptcha.svelte

```diff
@@ -12,25 +12,31 @@
   declare var hcaptcha: any;
 
   export enum CaptchaTheme {
-    DARK = "dark",
-    LIGHT = "light",
+    DARK = 'dark',
+    LIGHT = 'light',
   }
 </script>
 
 <script lang="ts">
-  import { onDestroy, createEventDispatcher, onMount } from "svelte";
+  import { onDestroy, createEventDispatcher, onMount } from 'svelte';
+  import { browser } from '$app/env';
 
   const dispatch = createEventDispatcher();
 
-  export let sitekey = window.sitekey || undefined;
-  export let apihost;
-  export let hl;
-  export let reCaptchaCompat;
-  export let theme;
+  export let sitekey: string = window?.sitekey || undefined;
```

Comment:
When I tested this inside a SvelteKit app, I ran into problems with `window is not defined` (even with the nullish coalescing operator you've added).

My suggestion here is;
1. Leave `sitekey` `undefined` i.e. `export let sitekey: string;`
2. Define it in the `onMount()` callback, dependent on `browser` i.e. 

```
onMount(() => {
  if (browser && !sitekey) sitekey = window.sitekey;
  ...
```

That seems to solve the problem for me. What do you reckon?

Reply:
Yep, works great 👍